data/types/basal/suppressed.go

```diff
@@ -56,6 +58,8 @@ func (s *Suppressed) Validate(validator data.Validator, allowedDeliveryTypes []s
 			}
 		}
 	}
+
+	// validator.ValidateInterfaceArray("annotations", s.Annotations)    // TODO: Any validations? Optional? Size?
```

Comment:
did you mean to leave this? why not just the todo if its a note

Reply:
It is a TODO so that if/when we have a way to validate we remember to add it here.